themes/default/content/docs/intro/pulumi-service/team-access-tokens.md

```diff
@@ -0,0 +1,129 @@
+
+Collectively Organization Access Tokens and Team Access Tokens are referred to as "machine tokens", which are not owned by a real user in your organization. This distinguishes them from Personal Access Tokens.
+
+Team Access Tokens are available on Enterprise and Business Critical subscriptions, as well as trials.
+
+Team Access Tokens provide several benefits over Organization and Personal Access Tokens:
+
+* They can be managed by Team Admins in addition to Organization Admins, allowing more users in your organization to leverage machine tokens.
+* They support user-independent usage in your CI integrations while having less privileged scope to other stacks in your organization.
+
+## Creating a Team Access Token
+
+Navigate to your organization, then:
+
+1. Select **Teams**.
+1. Select the team you would like to attach the token to.
+1. Scroll to **Access Tokens**.
+
+Team Access Token, like with Organization Access Tokens must have a name that is unique among all Organization Access Tokens assigned to it. This allows tokens taking operations on behalf of your organization to be identifiable in the event that one is compromised.
+
+This name must be unique even among deleted tokens, in order to maintain the integrity of Audit Log Events which persist the token’s name as part of the event (see below). Any other Organization Admin, or a Team Admin for the associated team, can delete the token, since they are managed by the team and not by a user.
+
+Creation of any Team Access Token, and the user who performed it, is logged as an Audit Log Event.
+
+## Viewing Team Access Tokens
+
+Team Access Tokens are viewed by going to the associated team's page within your respective organization, and scrolling to the Team Access Tokens card. Only Organization Admins and Team Admins for this team will see this card.
+
+## Deleting a Team Access Token
+
+A Team Access Token can be deleted by any Team Admin of this team, or any Organization Admin at any time. No other member types of the organization are able to delete a Team Access Token.
+
+From the organization's homepage, follow the same steps as for all other Access Token types:
+
+1. Select **Teams**.
+1. Navigate to the desired team.
+1. Scroll to the Team Access Tokens card.
+1. Select the ellipsis button.
+1. Choose **Delete token**. You will be prompted in a dialog to confirm your choice.
+
+If you choose to delete a token, its access will immediately be revoked and all further operations using it will fail as unauthorized. It's name will remain reserved for your organization to preserve its uniqueness in Audit Log Events for any operations that it carried out.
```

Comment:
```suggestion
If you choose to delete a token, its access will immediately be revoked and all further operations using it will fail as unauthorized. Its name will remain reserved for your organization to preserve its uniqueness in Audit Log Events for any operations that it carried out.
```

Nit: Also maybe mention that you already talked about this concept above?